functests/4_latency/latency.go

```diff
@@ -245,6 +246,18 @@ var _ = Describe("[performance] Latency Test", func() {
 	})
 })
 
+/*
+DescribeTable("Test different scenarios",func(lat_test_delay int, lat_test_run bool, runtime string, max_latency int){
+		os.Setenv("LATENCY_TEST_DELAY",lat_test_delay)
+		os.Setenv("LATENCY_TEST_RUN",lat_test_run)
+		os.Setenv("LATENCY_TEST_RUNTIME",runtime)
+		os.Setenv("MAXIMUM_LATENCY",max_latency)
+		init()
```

Comment:
this may be legal (= compiler doesn't complain) but it's almost always a bad idea. I'd just extract the code which set the globals in a function you can call in the latency tests proper - so just call it in `init` and in the test' tests

Reply:
Yeah I agree.This was just one of my retries, but I'll consider using your idea